File: Courzelo-backend-LMS/Courzelo/src/main/java/org/example/courzelo/dto/responses/institution/InstitutionUserResponse.java
package org.example.courzelo.dto.responses.institution;

import lombok.Builder;
import lombok.Data;
import org.example.courzelo.models.institution.Timeslot;

import java.util.List;

@Data
@Builder
public class InstitutionUserResponse {
    private String email;
    private String name;
    private String lastname;
    private List<String> roles;
    private String country;
    private String gender;
    private List<String> skills;
    private List<InstitutionTimeSlot> disponibilitySlots;
}

File: courzelo-dashboard-LMS-main/src/app/shared/models/institution/InstitutionUserResponse.ts
import {InstitutionTimeSlot} from './InstitutionTimeSlot';

export interface InstitutionUserResponse {
    email: string;
    name: string;
    lastname: string;
    roles: string[];
    country: string;
    gender: string;
    skills: string[];
    disponibilitySlots: InstitutionTimeSlot[];
}
